Linkedlist/reverse k nodes in ll.CPP

```diff
@@ -0,0 +1,69 @@
+/**

```

Comment:
@yash-rana23 Please update the code with **Problem Statement** and **Sample Test Case**. Code is directly from `Leetcode submission`. Update the PR so that it can be merged. 👨🏻‍💻